src/test/scala/sigmastate/serialization/generators/ValueGenerators.scala

```diff
@@ -209,4 +191,33 @@ trait ValueGenerators extends TypeGenerators {
       taggedVar(Arbitrary(sTupleGen(2, 10)))
     ))
   } yield Tuple(values)
+
+  val ergoBoxGen: Gen[ErgoBox] = for {
+    l <- arbLong.arbitrary
+    p <- proveDlogGen
+    b <- Gen.oneOf(TrueLeaf, FalseLeaf, p)
+    tId <- Gen.listOfN(32, arbByte.arbitrary)
+    boxId <- arbShort.arbitrary
+    regNum <- Gen.chooseNum[Byte](0, ErgoBox.nonMandatoryRegistersCount)
```

Comment:
@kushti this breaks an assumption in DataSerializer for SBox that all registers are densely packed.